src/functions/Mock.ps1

```diff
@@ -1038,6 +1038,7 @@ function Invoke-InMockScope {
 }
 
 function Test-ParameterFilter {
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute('Pester.BuildAnalyzerRules\Measure-SafeComands', 'Set-StrictMode', Justification = 'Used in mock-wrapper that runs in different context.')]
```

Comment:
We should pass command reference to the Set-Strict mode via parameter and call it from that.

```powershell
# when building mock parameters:
$parameters = @{
     [...]
     Set_StrictMode = $SafeCommands['Set-StrictMode']
}

# Then when we need to call it:
& $______mock_parameters.Set_StrictMode ...
```

Reply:
You don't have to solve this in this PR. Just pointing out that it is possible.

Reply:
Good point. Didn't cause any further failures in tests locally, so added it. 👍 
Now we just need to know why CI fails on PSv3 and PSv4